packages/contracts-bedrock/test/cannon/MIPS2.t.sol

```diff
@@ -1178,8 +1178,9 @@ contract MIPS2_Test is CommonTest {
     )
         public
     {
-        vm.assume(_wakeup != sys.FUTEX_EMPTY_ADDR);
-        vm.assume(_wakeup != _futexAddr);
+        // -2 incase _wakeup == _futexAddr and _wakeup needs to be incremented
+        _wakeup = uint32(_bound(_wakeup, 0, sys.FUTEX_EMPTY_ADDR - 2));
```

Comment:
Same as above

Reply:
FUTEX_EMPTY_ADDR is type(uint32).max that's why I used bound here